.changelog/637d44ac73834b409351f2497d3eca97.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "637d44ac-7383-4b40-9351-f2497d3eca97",
+    "type": "feature",
+    "collapse": true,
```

Comment:
not a big deal, but this should be false

Reply:
Just to collapse in case multiple module is displayed